src/components/map/map.tsx

```diff
@@ -50,10 +50,16 @@ export class Map implements IComponent {
    */
```

Comment:
Let's remove this commit from this PR so we don't mix the new gauge control with a fix on the map control.

Reply:
I make the changes and include the file by mistake... 😓 
I revert the changes in the commit b8bcfedbb990f986a4b074127c3ee8433026d7d2